### PR TITLE
Improve nodes articles

### PR DESCRIPTION
**Pull request by Hugoo**
- removed the part with IP as it is now automatically set
- cleaned the "testnet" node part as it was 90% the same as mainnet page
- various minor other improvements

### CODE REVIEW

File: docs/networks/mainnet/running-a-node.md [hunk 1]
```diff
@@ -71,85 +80,21 @@ Initialize the working folder, which will download the LUKSO network configurati
...
-
-Exchange the following sample addresses:
-
-```text title="[PATH_TO_NODE_WORKING_DIRECTORY]/configs/mainnet/lighthouse/lighthouse.toml"
-listen-address = "0.0.0.0"
-enr-address = "0.0.0.0"
-```
-
-With your own public IP addresses:
+After the initialization is successful, we must download the blockchain clients, which will be managed from the CLI under the hood. They will install globally, need superuser permissions, and are set as default clients within your working directories configuration file. You will be asked which clients you want to download and install during the setup.
```
Comment (biancabuzea200) [inline docs/networks/mainnet/running-a-node.md:91]:
```suggestion
After the initialization is successful, you must download the blockchain clients, which will be managed from the CLI under the hood. They will be installed globally, need superuser permissions, and are set as default clients within your working directories configuration file. You will be asked which clients you want to download and install during the setup.
```

File: docs/networks/mainnet/running-a-node.md [hunk 2]
```diff
@@ -39,9 +48,9 @@ LUKSO has partnered with [Dappnode](https://dappnode.com/) ([announcement](https
...
+The [LUKSO CLI](https://github.com/lukso-network/tools-lukso-cli) is a command line tool to install, manage and set up nodes and validators of different clients for the LUKSO blockchain. It provides simple and unified commands to interact with your node and runs natively on your operating system. The LUKSO CLI is officially supported for Ubuntu, Debian and macOS running on either x86 Intel and AMD Processors or Single Board ARM Computers.
 
-You can use the [LUKSO CLI](https://github.com/lukso-network/tools-lukso-cli) to prepare and run your LUKSO node. For further information, you can check out the official [LUKSO CLI Documentation](https://github.com/lukso-network/tools-lukso-cli/). It is not limiting yourself to running any supported clients, as all configurations and flags can be passed down.
+For further information, you can check out the official [LUKSO CLI Documentation](https://github.com/lukso-network/tools-lukso-cli/). It is not limiting yourself to running any supported clients, as all configurations and flags can be passed down.
```
Comment (biancabuzea200) [inline docs/networks/mainnet/running-a-node.md:53]:
```suggestion
For further information, you can check out the official [LUKSO CLI Documentation](https://github.com/lukso-network/tools-lukso-cli/). It does not stop you from running any supported clients, all configurations and flags can be passed down.
```

**Review by JeneaVranceanu — APPROVED** [review]
LGTM

File: docs/networks/testnet/running-a-node.md [hunk 3]
```diff
@@ -137,13 +72,5 @@ Ask your question in the validators channel on the [official LUKSO Discord serve
 
 ## Further Reads
 
-For extended help or advise for setting up your node beyond the LUKSO CLI, you can check out the following links.
```
Comment (biancabuzea200) [inline docs/networks/testnet/running-a-node.md:140]: why are we removing the community links?

Comment (Hugoo) [reply docs/networks/testnet/running-a-node.md:140]: They are already in the main page:

https://docs.lukso.tech/networks/mainnet/running-a-node

File: docs/networks/mainnet/running-a-node.md (hunk 2 above)
Comment (Hugoo) [reply docs/networks/mainnet/running-a-node.md:53]: Ty, changes applied here: https://github.com/lukso-network/docs/commit/fd979961f264d273ceca96c59688eb188dd6cac3

File: docs/networks/mainnet/running-a-node.md (hunk 1 above)
Comment (Hugoo) [reply docs/networks/mainnet/running-a-node.md:91]: Done here: https://github.com/lukso-network/docs/commit/fd979961f264d273ceca96c59688eb188dd6cac3

**Pull request merged**